pkg/schema/internal/rt.go

```diff
@@ -0,0 +1,7 @@
+package internal
```

Comment:
ah, `pkg/schema/internal/rt`, not `pkg/schema/internal`

Reply:
:D ooooooops

Reply:
no worries :)

Reply:
that means internal is in the package schema? i would have a cyclic import i think again.

Reply:
oh you mean pkg/schema/internal/internal.go?